docs_outdoornav_user_manual/features/foxglove_visualization.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Foxglove Visualization
+sidebar_label: Foxglove Visualization
+sidebar_position: 6
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+We have enabled Foxglove visualization as part of OutdoornNav, for improved debugging and data visualization and analysis.
+To access the Foxglove visualization interface for standard OutdoorNav configurations, navigate to the following 
+[URL](http://192.168.131.1/_/foxglove) (`platform_pc_ip_address`/\_/foxglove), where the _<platform_pc_ip_address>_ is
```

Comment:
Does 192.168.131.1 actually work here? I assume that only works if an Ethernet cable is connected, right?

Reply:
I guess this is pending feedback from Mike